### ISSUE TITLE
Pushing to fork: "error: failed to push some refs to"

### ISSUE THREAD

**Issue by sandertan** [git-lfs/git-lfs#3386]
I'm experiencing an issue with pushing a branch to a fork. When I push to the source repository I can push without issues. But when pushing the same branch to a fork, this results in: 
```
...... : no such file or directory 
error: failed to push some refs to 'https.........'
```

This issue can be reproduced by running the following:
```
# Clone the source repository
git clone https://github.com/cBioPortal/datahub.git
cd datahub

# Create your own fork of this repo and add it as remote
git remote add sandertan https://github.com/sandertan/datahub.git

# Create test branch and files
git checkout -b 'master_test'
touch test_file.txt
git add test_file.txt
git commit -m 'Add test file'

# Push this branch your own fork
git push sandertan master_test
```
This results in:
```
$ git push sandertan master_test
open /Users/sander/Data/datahub/public/prad_su2c_2015/data_mutations_misc.txt: no such file or directory                                                                                                                                                                      
error: failed to push some refs to 'https://github.com/sandertan/datahub.git'
```

Is there something wrong with how these files were added to git lfs on the source repository, or is there an issue with my git-lfs configuration?

```
$ git lfs env
git-lfs/2.6.0 (GitHub; darwin amd64; go 1.11.2)
git version 2.16.1

Endpoint=https://github.com/cBioPortal/datahub.git/info/lfs (auth=none)
Endpoint (sandertan)=https://github.com/sandertan/datahub.git/info/lfs (auth=basic)
LocalWorkingDir=/Users/sander/Data/datahub
LocalGitDir=/Users/sander/Data/datahub/.git
LocalGitStorageDir=/Users/sander/Data/datahub/.git
LocalMediaDir=/Users/sander/Data/datahub/.git/lfs/objects
LocalReferenceDirs=
TempDir=/Users/sander/Data/datahub/.git/lfs/tmp
ConcurrentTransfers=3
TusTransfers=false
BasicTransfersOnly=false
SkipDownloadErrors=false
FetchRecentAlways=false
FetchRecentRefsDays=7
FetchRecentCommitsDays=0
FetchRecentRefsIncludeRemotes=true
PruneOffsetDays=3
PruneVerifyRemoteAlways=false
PruneRemoteName=origin
LfsStorageDir=/Users/sander/Data/datahub/.git/lfs
AccessDownload=none
AccessUpload=none
DownloadTransfers=basic
UploadTransfers=basic
git config filter.lfs.process = "git-lfs filter-process --skip"
git config filter.lfs.smudge = "git-lfs smudge --skip -- %f"
git config filter.lfs.clean = "git-lfs clean -- %f
```

Some context on the contents of this repository: 
On cBioPortal Datahub (https://github.com/cBioPortal/datahub) we collect and curate published clinical and genomics data for cBioPortal (www.cbioportal.org), which is an open source web application for visualization, analysis and download of large-scale cancer genomics data sets. We are using git lfs for version control of the larger data files.

**Comment by bk2204** [comment]
Hey, thanks for writing in, and sorry that you're having trouble.  I think this issue is the same as #1113.  However, I appreciate that you've provided a very helpful set of reproduction steps, which will let us figure out what's going on and fix it.

I hope to have some time to look at this issue (and #1113) over the next week if local holidays don't interfere.

**Comment by bk2204** [comment]
I believe this issue has been fixed with #3398, so I'm going to close this now.
